File: api/src/main/java/com/hirrao/javaee/entity/Drug.java
package com.hirrao.javaee.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class Drug {
    @TableId(value = "drugId")
    //@TableField("drugId")
    private Long drugId;
    private Long uid;
    @TableField("drugName")
    private String drugName;
    private String frequency;
    private String unit;
    private Float dosage;
    @TableField("isActive")
    private int isActive;
}

File: api/src/main/kotlin/cn/hirrao/javaee/entity/Drug.kt
package cn.hirrao.javaee.entity

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId

data class Drug(
    @TableId(value = "drugId") //@TableField("drugId")
    val drugId: Long? = null,
    val uid: Long? = null,

    @TableField("drugName") private val drugName: String? = null,
    val frequency: String? = null,
    val unit: String? = null,
    val dosage: Float? = null,
    @TableField("isActive") val isActive: Int = 0,
)